webpack.base.config.js

```diff
@@ -77,47 +90,61 @@ module.exports = {
     rules: [
       {
         test: /\.vue$/,
-        loader: 'vue-loader',
-        include: [path.resolve(__dirname, 'app/components'), path.resolve(__dirname, 'updater')],
-        options: {
-          esModule: true,
-          transformToRequire: {
-            video: 'src',
-            source: 'src',
+        use: [
+          {
+            loader: 'vue-loader',
+            options: {
+              esModule: true,
+              transformToRequire: {
+                video: 'src',
+                source: 'src',
+              },
+            },
           },
-          loaders: { ts: 'awesome-typescript-loader' },
-        },
+        ],
+        include: [path.resolve(__dirname, 'app/components'), path.resolve(__dirname, 'updater')],
       },
       {
         test: /\.ts$/,
-        loader: 'awesome-typescript-loader',
-        options: { useCache: true, reportFiles: ['app/**/*.ts'] },
-        exclude: /node_modules|vue\/src/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'ts-loader',
+          options: {
+            reportFiles: tsFiles,
+            compilerOptions: {
+              strictNullChecks: !!process.env.SLOBS_STRICT_NULLS,
+            },
+          },
+        },
       },
       {
         test: /\.tsx$/,
         include: path.resolve(__dirname, 'app/components'),
-        loader: [
+        use: [
           'babel-loader',
```

Comment:
not actually sure we need babel-loader now that our target isn't es5, could boost build times a bit if we are able to get rid of this step

Reply:
Ah got it.  I thought TS compiler was translating from TSX => JSX and then babel was translating JSX => JS.  I can check if that's actually the case.

Reply:
oh that might be the case actually, not sure i haven't taken a look in a while